system-tests/tests/smoke/cre/por_test.go

```diff
@@ -483,7 +531,11 @@ func setupPoRTestEnvironment(
 		deployerPrivateKey:      universalSetupOutput.BlockchainOutput.DeployerPrivateKey,
 		creCLIAbsPath:           creCLIAbsPath,
 		creCLIsettingsFile:      creCLISettingsFile,
+		readTargetName:          fmt.Sprintf("read-contract-%s-%d@1.0.0", "evm", universalSetupOutput.BlockchainOutput.ChainID),
```

Comment:
ideally we'd use the same function that core node uses, when registering the capability, so that they do not drift away. I did something like that for `write-target`. Do you think it would be possible here?